x/ecocredit/client/tx.go

```diff
@@ -278,7 +278,7 @@ func TxSendCmd() *cobra.Command {
 Parameters:
   recipient: recipient address
   credits:   YAML encoded credit list. Note: numerical values must be written in strings.
-             eg: '[{batch_denom: "100/2", tradable_amount: "5", retired_amount: "0", retirement_location: "YY-ZZ 12345"}]'
+             eg: '[{batch_denom: "C01-20210101-20220101-001", tradable_amount: "5", retired_amount: "0", retirement_location: "YY-ZZ 12345"}]'
```

Comment:
nit: do we need to have validation to not accept `/` or any other special chars in the denom?

Reply:
We already require the denom to conform to `FormatDenom` (i.e. `%s-%s-%s-%03d`). We will not accept `/` or any other special chars in the denom. This change here is only updating the examples in the CLI descriptions to do so.

Reply:
We already require the denom to conform to `FormatDenom` (i.e. `%s-%s-%s-%03d`). We will not accept `/` or any other special chars in the denom. This change here is only updating the examples in the CLI descriptions.

Reply:
Looks like we are not using `ValidateDenom` in `MsgSend` and `MsgRetire`.

Reply:
Opened #551.